modules/apps/headless/headless-builder/headless-builder-test/src/testFunctional/paths/APIBuilder.path

```diff
@@ -190,6 +205,11 @@
 	<td>//button[(@class='view-related-objects btn btn-secondary') and contains(.,'View Related Objects')]</td>
 	<td></td>
 </tr>
+<tr>
+	<td>REMOVE_VIEW_RELATED_OBJECTS</td>
+	<td>//div[@class='related-objects-results-bar-container']//button[@aria-label='Back']</td>
+	<td></td>
```

Comment:
you can try to use 'Button#ANY_WITH_ARIA_LABEL' to replace it, could you take a look at it? thanks!